main/src/ActualCoffeeMachine.java

```diff
@@ -0,0 +1,119 @@
+import java.util.Scanner;
+
+public class ActualCoffeeMachine {
+    public static void main(String[] args) {
```

Comment:
Can you split main method to more than one small methods? with comments

Reply:
Hayk i have done your requirements